db/seeds.dev.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+def create_rule_results(test_results, passing)
+  test_results.each do |tr|
+    rule_result_columns = tr.profile.rules.pluck(:id).map do |rule_id|
+      [rule_id, tr.id, tr.host_id, passing ? 'pass' : %w[pass pass fail].sample]
+    end
+
+    RuleResult.import(
+      %i[rule_id test_result_id host_id result], rule_result_columns
+    )
+
+    tr.update!(score: tr.rule_results.passed.count / tr.rule_results.count.to_f * 100)
+  end
+end
+
+def create_test_results(profile, host, passing)
+  test_result_count = ENV['TEST_RESULT_COUNT'] || rand(2..3)
+
+  test_results = FactoryBot.create_list(
+    :test_result,
+    test_result_count,
+    profile: profile,
+    host: host,
+    supported: SupportedSsg.supported?(
+      ssg_version: profile.ssg_version,
+      os_major_version: host.os_major_version,
+      os_minor_version: host.os_minor_version
+    )
+  )
+
+  create_rule_results(test_results, passing)
+  profile.calculate_score!
+end
+
+def find_or_create_profile(policy, canonical_profile, os_minor_version)
+  profile = policy.reload.profiles.find_by(os_minor_version: os_minor_version)
+  profile || canonical_profile.clone_to(
+    policy: policy,
+    account: policy.account,
+    os_minor_version: os_minor_version.to_s
+  )
+end
+
+def create_policy(account, canonical_profile, hosts)
+  profile = Profile.new(
+    parent_profile_id: canonical_profile.id,
+    account: account
+  ).fill_from_parent
+
+  policy = Policy.new(
+    profile.slice(:name, :description, :compliance_threshold, :account)
+  )
+
+  policy.save!
+  profile.update!(policy: policy)
+  policy.update_hosts(hosts.pluck(:id))
+
+  policy
+end
+
+logger = Logger.new(STDOUT)
+account_count = ENV['ACCOUNT_COUNT'] || 3
+
+FactoryBot.create_list(:account, account_count)
+logger.info("Generated #{account_count} accounts.")
+
+Account.find_each do |account|
+  host_count = ENV['HOST_COUNT'] || rand(2..3)
+
+  host_ids = SupportedSsg.all.map { |ssg| [ssg.os_major_version, ssg.os_minor_version] }.uniq.flat_map do |version|
+    FactoryBot.create_list(
+      :host,
+      host_count,
+      account: account.account_number,
+      os_version_arr: version
+    ).map(&:id)
+  end
+
+  logger.info("Generated #{host_count} hosts for acc #{account.account_number}")
```

Comment:
Got hosts generated for the same account twice. I guess it is because I've run it second time.
```
I, [2021-06-29T14:23:47.581223 #174581]  INFO -- : Generated 3 hosts for acc 00002
I, [2021-06-29T14:24:42.808221 #174581]  INFO -- : Generated 15 policies for account 00002
I, [2021-06-29T14:24:43.062831 #174581]  INFO -- : Generated 3 hosts for acc 00002
```